src/app/server/Server.cpp

```diff
@@ -180,6 +181,25 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     }
     else
     {
+        DeviceLayer::ConfigurationMgr().LogDeviceConfig();
```

Comment:
I think most applications print this information as well as the onboarding codes in their `AppTask::Init` (or other application init functions). Should we remove all that code if it's now moved to the common server code?

Reply:
Overall, I don't think any logging of QR code should be done by common SDK code. That is a demo detail. Real products likely won't even know their passcode, since they can be configured with just SPAKE2+ verifier computed from passcode, so they can avoid computing the verifier.

Reply:
See commit message: it's a matter of whether it's more convenient for that info to be at the start of the log or right at the point when the app is waiting for you to talk to it.  The answer might be "both" depending, so I left it both places.

Reply:
> Overall, I don't think any logging of QR code should be done by common SDK code.

That is fair.  We could put the logging in the per-example mains, but after the server Init() call, perhaps, so it's visible at the tail of the log and easier to find there?  Alternately, we could just fix the incorrect value being logged in the "linux" impl and drop the rest of the changes and people should know to read the head of the log for commissioning info.